components/region_cache_memory_engine/src/background.rs

```diff
@@ -513,6 +531,10 @@ impl Runnable for BackgroundRunner {
     fn run(&mut self, task: Self::Task) {
         match task {
             BackgroundTask::Gc(t) => {
+                info!(
```

Comment:
I think it will be too verbose. Maybe debug! is better.

Reply:
I think it's fine. It will be printed only once every three minutes by default.